public/app/features/search/components/ManageDashboardsNew.tsx

```diff
@@ -43,9 +43,9 @@ export const ManageDashboardsNew = React.memo(({ folder }: Props) => {
     <>
       <div className={cx(styles.actionBar, 'page-action-bar')}>
         <div className={cx(styles.inputWrapper, 'gf-form gf-form--grow m-r-2')}>
-          <Input
+          <FilterInput
```

Comment:
Actually we need to pass in `escapeRegex={false}` to this component or the search will not be correct.